build.gradle

```diff
@@ -28,20 +28,25 @@ ext {
 		!it.name.equals("spring-build-src") && !it.name.equals("spring-framework-bom")
 	}
 
+	activationApiVersion = "1.2.0"
 	aspectjVersion       = "1.9.2"
 	freemarkerVersion    = "2.3.28"
 	groovyVersion        = "2.5.5"
 	hsqldbVersion        = "2.4.1"
 	jackson2Version      = "2.9.8"
+	jaxbApiVersion       = "2.3.1"
 	jettyVersion         = "9.4.14.v20181114"
+	junit4Version        = "4.12"
 	junit5Version        = "5.3.2"
+	jspApiVersion        = "2.3.2-b02"
```

Comment:
This should be moved ideally before `junit4Version` to respect the current order.

Reply:
Sorry. I update it.